wombat-admin/src/app/page.tsx

```diff
@@ -77,6 +80,49 @@ export default function Home() {
 
   const closeViewImageModal = (): void => setSelectedRequest(null)
 
+  // Function to show the status message modal and set a timer to hide it after 5 seconds
+  const showStatusMessage = () => {
+    setStatusMessage(true);
+    setTimeout(() => {
+      setStatusMessage(false);
+    }, 2000);
+  };
+
+  // const updateRequestStatus = async (requestId: string, approvalStatus: 'APPROVED' | 'DENIED') => {
```

Comment:
I've mentioned this before, but PLEASE don't commit commented out code. If this was needed before, we would have it in the git history.